src/main/java/com/iemr/hwc/data/foetalmonitor/FoetalMonitor.java

```diff
@@ -276,20 +276,20 @@ public void setPartnerName(String partnerName) {
 		this.partnerName = partnerName;
 	}
 
-	public String getFetosenseMotherID() {
+	public String getFoetalMonitorMotherID() {
 		return cmMotherId;
 	}
 
-	public void setFetosenseMotherID(String fetosenseMotherID) {
-		this.cmMotherId = fetosenseMotherID;
+	public void setFoetalMonitorMotherID(String foetalMonitorMotherID) {
+		this.cmMotherId = foetalMonitorMotherID;
```

Comment:
@swetatech24 @Anjana2310 Can we rename foetalMonitoryMotherID to just montherId?